code/modules/worldgen/GenerateMining.dm

```diff
@@ -522,6 +522,13 @@ var/list/miningModifiersUsed = list()//Assoc list, type:times used
 		prefabPath = "assets/maps/prefabs/prefab_dans_asteroid.dmm"
 		prefabSizeX = 37
 		prefabSizeY = 48
+
+	drug_den // A highly cozy hideout in space; take out the stress - eat some mice sandwiches.
+		maxNum = 1
+		probability = 100
```

Comment:
100%? I think it should be less.

Reply:
Already changed that. XD
My bad - forgot to lower it after testing. It is now at 40.